Integrations/integration-AMP.yml

```diff
@@ -174,30 +174,32 @@ script:
         return undefined;
     }
 
+    function executeCommand(command, cmdArgs) {
```

Comment:
executeCommand is saved name in demisto. We have builtin command called executeCommand.
so change this function name to something else